README.md

```diff
@@ -17,21 +17,23 @@ our [documentation](https://www.elastic.co/guide/en/security/master/get-started-
 
 ___
 
-## Development
+## Table of contents
 
-### Table of contents
-
-- [Prerequisites](#prerequisites)
-- [Deploying Cloudbeat as a process](#deploying-cloudbeat)
-    - [Unmanaged Kubernetes](#clean-up)
-    - [Amazon Elastic Kubernetes Service (EKS)](#amazon-elastic-kubernetes-service-(EKS))
+- [Local Deployment](#local-deployment)
+    - [Unmanaged Kubernetes](#deploying-cloudbeat-as-a-process)
+      - [Self-Managed Kubernetes (Kind)](#self-managed-kubernetes)
+      - [Amazon Elastic Kubernetes Service (EKS)](#amazon-elastic-kubernetes-service-(EKS))
```

Comment:
Should EKS be part of the Unmanaged Kubernetes? Should it be Agentless cloudbeat/cloudbeat standalone/deploy as a process?

Reply:
Did you remove the `[Prerequisites](#prerequisites)` on purpose?

Reply:
EKS link is broken